app/services/github_client/organization.rb

```diff
@@ -1,9 +1,14 @@
 module GithubClient
   class Organization < GithubClient::Base
     MAX_REPOSITORIES_PER_PAGE = 100
+    MAX_USERS_PER_PAGE = 30
 
     def repositories
       get_all_paginated_items('orgs/rootstrap/repos', MAX_REPOSITORIES_PER_PAGE)
     end
+
+    def members
```

Comment:
I'd add a test for this method in the file `spec/services/github_client/organization_spec.rb`